src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,14 @@
+server.port = 8080
+
+spring.datasource.url=jdbc:postgresql://localhost:5432/argos_dev_db
+spring.datasource.username=user
```

Comment:
please use environment variables here something like the following:  
spring.datasource.username:  "${DB_USER}"
 spring.datasource.password:    "${DB_PASSWORD}"

Reply:
Got it!